node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,13 @@
+import { readFile } from 'node:fs/promises';
+
+async function readAny() {
+  try {
+    const filePath = process.argv[2];
+    const contents = await readFile(`./${filePath}`, { encoding: 'utf-8' });
```

Comment:
You could just pass `filePath`, and then expect the user to enter the correct path on the command line.